### PR TITLE
[ricos-plugins] internal: external, mobile & inline toolbars trigger buttonClick event

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/ricos-types/web/src/toolbarEnums.ts [hunk 1]
```diff
@@ -54,6 +54,8 @@ export enum ToolbarType {
   TEXT = 'ALL-TEXT-TOOLBARS',
   SHORTCUT = 'SHORTCUT',
   LINK = 'LINK',
+  EXTERNAL = 'EXTERNAL',
```
Comment (lxgreen) [inline packages/ricos-types/web/src/toolbarEnums.ts:57]: it is bad idea to use enums

Comment (lxgreen) [reply packages/ricos-types/web/src/toolbarEnums.ts:57]: they are either weird non-type objects that impact bundle size like bitches (non-const enums), or non-importable strings (const enums). Please use sum type instead:
```
SIDE | MOBILE | ...
```

Comment (lxgreen) [reply packages/ricos-types/web/src/toolbarEnums.ts:57]: this whole file should be converted to sum types

Comment (lxgreen) [reply packages/ricos-types/web/src/toolbarEnums.ts:57]: we use eums due to ts-proto and wix-proto-gen conversions, and they are banned in other places.

Comment (lxgreen) [reply packages/ricos-types/web/src/toolbarEnums.ts:57]: currently we have many issues with schema-derived types due to enums. e.g. Node_Type cannot be imported outside our monorepo

**Review by lxgreen — CHANGES_REQUESTED** [review]

File: packages/ricos-types/web/src/toolbarEnums.ts (hunk 1 above)
Comment (maimonav) [reply packages/ricos-types/web/src/toolbarEnums.ts:57]: This is used by consumers, if we change this file it might break them, no?

Comment (lxgreen) [reply packages/ricos-types/web/src/toolbarEnums.ts:57]: how is this used by consumers? it is ricos-types => tiptap

Comment (lxgreen) [reply packages/ricos-types/web/src/toolbarEnums.ts:57]: oh it is moved from one of commons?
shit

Comment (lxgreen) [reply packages/ricos-types/web/src/toolbarEnums.ts:57]: nevermind then. we'll take care of this later

**Pull request merged**